server/src/stateManagement.mjs

```diff
@@ -86,61 +86,13 @@ function addUser(userId) {
     logger.info(`Cannot add user ${userId}, already exists`);
     return {isSuccess: false, msg : `Cannot add user, already exists`};
   }
-
+  //To check whether the user,appId already exist 
+  let userExist = isUserExist(users,userId)
+  if((!userExist.isSuccess)){
```

Comment:
1) Extra brackets
2) Let's also check to be sure userExist is there as well. This could throw an exception.

Reply:
Addressed